.github/workflows/ci.yml

```diff
@@ -14,22 +14,14 @@ jobs:
       matrix:
         include:
           - pair:
-              elixir: 1.13
-              otp: 24.3
+              elixir: 1.13.x
+              otp: 22.x
```

Comment:
Minimum support OTP for Elixir 1.13.x is OTP 22, see https://hexdocs.pm/elixir/1.13/compatibility-and-deprecations.html#compatibility-between-elixir-and-erlang-otp